File: 프로그래머스/0/181874. A 강조하기/A 강조하기.java
class Solution {
    public String solution(String myString) {
        
        return myString.toLowerCase().replaceAll("a", "A");
    }
}

// 다른 풀이
return myString.chars()
               .mapToObj(c -> String.valueOf((char) (c == 97 || c == 65 ? Character.toUpperCase(c) : Character.toLowerCase(c))))
               .collect(Collectors.joining());

File: 프로그래머스/0/181874. A 강조하기/A 강조하기.py
def solution(myString):
    
    return myString.lower().replace('a','A')

# 다른 풀이
answer = ''
for i in myString:
    if i == 'a' or i == 'A':
        answer += 'A'
    elif i.isupper():
        answer += i.lower()
    else:
        answer += i
return answer
